docs/articles/documentation/reference/test-api/testcontroller/setnativedialoghandler.md

```diff
@@ -26,18 +26,17 @@ Argument | Type   | Description
 `text`   | String | Text of the dialog message.
 `url`    | String | The URL of the page that invoked the dialog. Use it to determine if the dialog originated from the main window or an `<iframe>`.
 
-Once the handler is specified, it will be triggered each time a native dialog appears in the test whether it originates from the main window or an `<iframe>`.
-You can provide a new handler at any moment by calling `t.setNativeDialogHandler` once again.
+Once the handler is specified, it is triggered each time a native dialog appears in the test whether it originates from the main window or an `<iframe>`.
```

Comment:
```suggestion
Once the handler is specified, it is triggered each time a native dialog appears in the test - whether it originates from the main window or an `<iframe>`.
```

Reply:
Yes, the dialog

An attempt:
> Once specified, the handler fires each time a native dialog appears in the test. The dialog can originate from the main window or an `<iframe>`.

Reply:
`Once specified, the handler fires each time a native dialog appears in the test. The dialog can originate from the main window or an <iframe>.`

Thank you. Please apply your new suggestion.

Reply:
Done